src/components/SearchBar.css

```
@@ -0,0 +1,42 @@
+.input {
+  width: 100%;
+  height: 5rem;
+  padding: 1% 10%;
+  border-radius: 8px;
+  margin-bottom: 6%;
+  background-color: #1f2a48;
+  border: none;
+}
+
+.input::placeholder {
+  font-size: 1.5rem;
+  font-family: "Space Mono", monospace;
+  color: #dcdde1;
+}
+
+.label {
+  position: relative;
+}
+
+.btn {
+  position: absolute;
+  right: 0;
+  top: 50%;
+  transform: translate(-10%, -50%);
+  font-size: 1.5rem;
+  font-family: "Space Mono", monospace;
+  padding: 1rem;
+  border-radius: 8px;
+  color: #dcdde1;
+  background-color: #184d93;
+  background-color: #0277f3;
```

```suggestion
  background-color: #0277f3;
  border: none:
```

To remove the current default border on the button. Blends better that way

---

Skipped that.. done